NRF24 message
---
bool succes
